src/internal/lokiutil/lokiutil.go

```diff
@@ -58,15 +74,15 @@ func QueryRange(ctx context.Context, c *loki.Client, queryStr string, from, thro
 		if err := forEachLine(*resp, func(t time.Time, line string) error {
 			from = t
 			nMsgs++
-			var entry struct {
-				Log    string    `json:"log"`
-				Stream string    `json:"stream"`
-				Time   time.Time `json:"time"`
-			}
+			entry := streamEntry{}
 			err := json.Unmarshal([]byte(line), &entry)
 			if err != nil {
 				return errors.EnsureStack(err)
 			}
+			entry, err = wrapEntryLog(entry)
```

Comment:
Rather than a misleadingly-named function call, i _think_ that this could just be legibly inlined. Something like `if err := json.Unmarshal(…); err != nil { if entry.log, err = json.Marshal(…); err != nil {…}}`?

Or maybe just renamed to something like `wrapIfNotJSON`. That name is kinda terrible though.

Reply:
As of now I renamed the function to `wrapLogIfNotJSON`, but like I said, I am happy to inline the function if we think the test is not needed or could be done some other way.